python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -1012,7 +1008,7 @@ def load(self, ids, load_backup=False):
         for this_id in ids:
 
             if this_id in self.incomplete_objects:
-                raise RepositoryError(self.repo, "Trying to re-load a corrupt repository id: %s" % this_id)
+                raise RepositoryError(self.registry, "Trying to re-load a corrupt repository id: %s" % this_id)
```

Comment:
Frankly something was better than the ```.repo``` argument here which was causing name errors. And there used to be 2 RepositoryError classes which added confusion.